tests/config.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+module.exports = {
+  GET_GIT_SHA_CMD: "git rev-parse HEAD",
+  GENERATE_PROJECTS: ["retail-app-demo", "retail-app-ext", "retail-app-no-ext"],
+  GENERATOR_CMD:
+    "node packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js --outputDir",
```

Comment:
if you want a true e2e test, then instead of using a path that's just designed for development i.e. `packages/pwa-kit-create-app/scripts/create-mobify-app-dev.js`, we could use verdaccio to run a local npm registry and use `npx @salesforce/pwa-kit-create-app` to generate the project, this is the command that real users use.

Reply:
maybe consider extracting the local npm registry verdaccio logic into it's own module for re-use.

Reply:
I've added a TODO to do this. I'll work on it towards the end while polishing code. Thanks! :D